src/client/Microsoft.Identity.Client/ITokenCache.cs

```diff
@@ -78,8 +78,10 @@ public partial interface ITokenCache
         /// want to call <see cref="ITokenCacheSerializer.DeserializeMsalV3(byte[], bool)"/></remarks>
 #if !SUPPORTS_CUSTOM_CACHE
         [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
-# endif
+#endif
+#pragma warning disable VSTHRD200 // Use "Async" suffix for async methods
```

Comment:
This rule is a combined version of two rules we used to have. UseAsyncSuffix and AvoidAsyncSuffix

Since this name is already in the public api, changing it is a breaking change. Thus, I am adding a suppression. 

https://github.com/microsoft/vs-threading/blob/main/doc/analyzers/VSTHRD200.md